cmd/fetchd/cmd/stakereconciliationmigrate.go

```diff
@@ -0,0 +1,306 @@
+package cmd
+
+import (
+	"encoding/csv"
+	"encoding/json"
+	"fmt"
+	"io"
+	"io/ioutil"
+	"os"
+	"strings"
+
+	"github.com/cosmos/cosmos-sdk/client"
+	"github.com/cosmos/cosmos-sdk/codec"
+	sdk "github.com/cosmos/cosmos-sdk/types"
+	"github.com/cosmos/cosmos-sdk/types/errors"
+	authtypes "github.com/cosmos/cosmos-sdk/x/auth/types"
+	banktypes "github.com/cosmos/cosmos-sdk/x/bank/types"
+	"github.com/spf13/cobra"
+	tmjson "github.com/tendermint/tendermint/libs/json"
+)
+
+type IneligibleRegistration struct {
+	NativeAddr string
+	Reason     error
+}
+
+func (i IneligibleRegistration) Print() {
+	fmt.Fprintf(os.Strerr, "%q ineligible for reason: %s\n", i.NativeAddr, i.Reason.Error())
```

Comment:
Ooops, I botched the previous suggestion!

```suggestion
	fmt.Fprintf(os.Stderr, "%q ineligible for reason: %s\n", i.NativeAddr, i.Reason.Error())
```